_posts/2016-04-03-presentation-materials.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Apache Beam Presentation Materials"
+date:   2016-04-03 11:00:00 -0700
+excerpt_separator: <!--more-->
+categories: beam capability
+authors:
+  - fjp
+  - takidau
+---
+
+Are you interested in giving a presentation about Apache Beam? Perhaps you want to talk about Apache Beam at a local Meetup or a convention. Excellent!  The Apache Beam community is excited to expand and grow the community. To help kickstart this process, we are excited to announce an initial set of [Apache Beam presentation materials]({{ site.baseurl }}/presentation-materials/) which anyone can use to give a presentation about Apache Beam.
+
+<!--more-->
+
+This is the first of many steps the Apache Beam community will take to create high quality presentation materials. The Beam community has worked together to ensure these materials are accurate, up-to-date, and relevant. This initial release includes the following presentation materials:
```

Comment:
How about just: 

As a community, we want to build a shared collection of high quality presentation materials. This initial set includes the following slide decks: